src/javascripts/legacy_app.js

```diff
@@ -1,8 +1,8 @@
 import BaseApp from 'base_app';
 
-// remove these if not needed
```

Comment:
how do we make it obvious to people who try to use them that they need to be uncommented?